_docs/guides/best-practices.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Best Practices 
+order: 004
+---
+# {{ page.title }}
+
+## Service Accounts
+When you use multiple service accounts with your Forseti Security deployments,
+you fulfill the security best practice of least privilege. Following are the
+scenarios for which it's best to use separate service accounts:
+
+ * **[Forseti Security Service Account](#forseti-security-service-account)**
+ (required): Used by all core modules of the program to provide basic
+ inventory, scanning, and enforcement actions.
+ * **[GSuite Groups Service Account](#gsuite-groups-service-account)**
```

Comment:
I disagree here, I think it's important to note that this account's access is scoped only the Groups API. That should be reflected in the name and in the docs that way.